src/PrestaShopBundle/Form/Admin/Product/ProductSpecificPrice.php

```diff
@@ -186,10 +186,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'label' => $this->translator->trans('Apply', array(), 'Admin.Actions'),
             'attr' => array('class' => 'btn-primary-outline js-save'),
         ))
-        ->add('cancel', 'Symfony\Component\Form\Extension\Core\Type\ResetType', array(
+        ->add('cancel', 'Symfony\Component\Form\Extension\Core\Type\ButtonType', array(
             'label' => $this->translator->trans('Cancel', array(), 'Admin.Actions'),
             'attr' => array('class' => 'btn-default-outline js-cancel'),
         ));
+        /*
+         * note: ResetType can't be used because the product page is wrapped inside a big form
```

Comment:
This is perfectly valid in PHP. More, this is a multiline comment and apply this comment 
on top of this class make the chances to be read close to 0.

I agree on the redundancy of ``note``, removed 👍